lib/src/main/java/io/strmprivacy/driver/client/AuthService.java

```diff
@@ -53,8 +53,8 @@ public AuthService(String clientId, String clientSecret, Config config) {
         }
 
         try {
-            this.authUri = new URI(config.getKeycloakUrl()).toString();
-            this.refreshUri = new URI(config.getKeycloakUrl()).toString();
+            this.authUri = new URI(String.format("%s://%s/%s", config.getKeycloakScheme(), config.getKeycloakHost(), config.getKeycloakEndpoint())).toString();
```

Comment:
The slash should be removed here for the path. That should be included in the path parameter.